<launch>
  <node name="PositionEstimator_Back" pkg="runway_ImageProc" type="PositionEstimator2" />
  <node name="GPS_PositionEstimator" pkg="runway_ImageProc" type="GPS_PositionEstimator" />

  <param name="back_showImages" type="bool" value="true" />
  <param name="back_uncertaintyPixels" type="int" value="10" />
  <param name="back_uncertaintyPixels" type="int" value="10" />
  <param name="back_thresholdValue" type="int" value="180" />
  <param name="back_cropTop" type="int" value="100" />
  <param name="back_cropBottom" type="int" value="1000" />
</launch>
